include/eve/module/real/math/function/regular/generic/sinhc.hpp

```diff
@@ -79,7 +79,7 @@ namespace eve::detail
 
         T x = abs(a0);
         auto lt1= is_less(x, one(eve::as<T>()));
-        std::size_t nb = nbtrue(lt1);
+        std::size_t nb = eve::count_true(lt1);
```

Comment:
signed unsigned?
nb maybe should be renamed?

Reply:
Fixed